doc/api/brotli.md

```diff
@@ -0,0 +1,491 @@
+
+Initializes an decoder with the given [options][].
+
+### Class Property: Decompress.codes
+<!-- YAML
+added: REPLACEME
+-->
+
+Error codes for decompression operations.
+More info can be found at [Error Codes for Decompression][].
+
+## brotli.constants
+<!-- YAML
+added: REPLACEME
+-->
+
+* {Object}
+
+Provides an object enumerating Brotli-related constants.
+More info can be found at [Constants][]
+
+## Convenience Methods
+
+<!--type=misc-->
+
+All of these take a [`Buffer`][], [`TypedArray`][], [`DataView`][],
+[`ArrayBuffer`][] or string as the first argument, an optional second argument
+to supply options to the `brotli` classes and will call the supplied callback
+with `callback(error, result)`.
+
+Every method has a `*Sync` counterpart, which accepts the same arguments, but
+without a callback.
+
+### brotli.compress(data[, options][, callback])
+<!-- YAML
+added: REPLACEME
+-->
+
+- `data` {Buffer|TypedArray|DataView|ArrayBuffer|string}
+- `options` {Object} an [options][] object.
+- `callback` {Function}
+
+Compress a chunk of data.
+If callback is omitted a {Promise} will be returned.
+
+### brotli.compressSync(data[, options])
+<!-- YAML
+added: REPLACEME
+-->
+
+- `data` {Buffer|TypedArray|DataView|ArrayBuffer|string}
+- `options` {Object} an [options][] object.
+
+Compress a chunk of data.
+
+### brotli.decompress(data[, options][, callback])
+<!-- YAML
+added: REPLACEME
+-->
+
+- `data` {Buffer|TypedArray|DataView|ArrayBuffer|string}
+- `options` {Object} an [options][] object.
+
+Decompress a chunk of data.
+If callback is omitted a {Promise} will be returned.
+
+### brotli.decompressSync(data[, options])
+<!-- YAML
+added: REPLACEME
+-->
+
+- `data` {Buffer|TypedArray|DataView|ArrayBuffer|string}
+- `options` {Object} an [options][] object.
+- `callback` {Function}
+
+Decompress a chunk of data.
+
+[`.flush()`]: #brotli_brotli_flush_callback
+[`.setParameter()`]: #brotli_brotli_setparameter_param_value
+[`Accept-Encoding`]: https://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html#sec14.3
+[`ArrayBuffer`]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/ArrayBuffer
+[`brotli.constants`]: #brotli_brotli_constants
+[`brotli/decode.h`]: https://github.com/google/brotli/blob/v1.0.4/c/include/brotli/decode.h
+[`brotli/encode.h`]: https://github.com/google/brotli/blob/v1.0.4/c/include/brotli/encode.h
+[`Buffer`]: buffer.html#buffer_class_buffer
+[`Compress.codes`]: #brotli_class_property_compress_codes
+[Constants]: #brotli_constants
+[`Content-Encoding`]: https://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html#sec14.11
+[`DataView`]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/DataView
+[`Decompress.codes`]: #brotli_class_property_decompress_codes
+[Error Codes for Compression]: #brotli_error_codes_for_compression
+[Error Codes for Decompression]: #brotli_error_codes_for_decompression
+[Parameters]: #brotli_parameters
+[options]: #brotli_options_object
```

Comment:
It seems the backticks should be left, otherwise, we have[ false linkification](https://github.com/Hackzzila/node/blob/72c2d4e864ad70286a3a9ef1a2e9404113e71065/doc/api/brotli.md#new-brotlicompressoptions) in the headings:

Reply:
Better than inconsistent/broken docs 😄